### PR TITLE
Added 06-rules markdown.

### PR DESCRIPTION
**Pull request by volbap**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [x] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [x] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description
- Added `06-rules.md`.
- Updated `index.yml`.

Thank you!

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/ios-swift/06-rules.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+let profile: A0UserProfile = ... // the user profile you get upon login
+guard let country = profile.extraInfo["country"] as? String else {
+    return // test failed
+}
+print("user country is: \(country)")
+```
+
+### Done!
+
+That's it. You've just experienced how to implement a basic rule. This is just one of all the cool things you can do with rules. Go ahead and create any rule that fits your needs.
```
Comment (tenmaster) [inline articles/native-platforms/ios-swift/06-rules.md]: I wouldn't write 3 times rules in the same paragraph. You can refer to them indirectly.
"This is just one of all the cool things you can do from now on"
"Go ahead and create any of them to fit your needs."

Comment (volbap) [inline articles/native-platforms/ios-swift/06-rules.md]: Agreed. @alemata check your tutorial too ^ hehe

**Comment by tenmaster** [comment]
+1 @ntotten

**Comment by mpaktiti** [comment]
Please remove the pending label once the PR is ready for merge.

**Comment by mpaktiti** [comment]
@pablolvillar Please rebase to resolve conflicts.

**Comment by volbap** [comment]
@mpaktiti done

File: articles/native-platforms/ios-swift/06-rules.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-swift',
+  pkgFilePath: 'iOS/basic-sample-swift/SwiftSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+Rules are one of the cool features of Auth0. The reason behind that coolness is their flexibility, which gives you the ability to extend what Auth0 has to offer. They are just JavaScript functions which work like middlewares. To see a detailed description, please refer to [the full documentation](https://auth0.com/docs/rules).
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/06-rules.md]: @pablolvillar We only use relative links. This `https://auth0.com/docs/rules` should be replaced with `/rules`.

File: articles/native-platforms/ios-swift/06-rules.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+### 1. Create a Rule
+
+To create a rule, just go to the [new rule page](https://manage.auth0.com/#/rules/new). You can create it from scratch or use an existing template. These templates are written by Auth0 team to assist you complete common tasks.
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/06-rules.md]: @pablolvillar We use a variable for the dashboard. This `https://manage.auth0.com/#/rules/new` should be `${uiURL}/#/rules/new`

**Comment by mpaktiti** [comment]
Seed project: https://github.com/auth0-samples/auth0-ios-swift-sample/pull/8

**Comment by volbap** [comment]
@mpaktiti Done with the fixes.

**Comment by mpaktiti** [comment]
Before deployment:
- [Seed project](https://github.com/auth0-samples/auth0-ios-swift-sample/pull/8) should be merged. The URL in this page should be updated (currently points to invalid sample).
- Sample is at auth0-samples repo, download zip functionality should be in place.
- `index.yml` should be checked again

**Comment by volbap** [comment]
@mpaktiti I changed the seed project link with a download button, following [these guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects). Let me know if it works, so I can apply it to the rest of the samples!

**Comment by volbap** [comment]
Closing this pull request. These changes will be included in a new pull request.

**Pull request closed without merging**